README.md

```diff
@@ -12,7 +12,13 @@
 <a href="http://mypy-lang.org"><img src="http://www.mypy-lang.org/static/mypy_badge.svg" alt="Checked with mypy"></a>
 </p>
 
-Atlas Download Tools is a toolbox to download atlas data.
+Atlas-Download-Tools - search, download, and prepare atlas data.
```

Comment:
I just noticed by looking at the preview of `README.md` that there's a bit of a duplication: the title of the README is already `Atlas-Download-Tools` and we repeat it in the next line again.

So maybe we can drop `Atlas-Download-Tools` and just write
```
Search, download, and prepare atlas data.
```